src/main/java/mate/academy/library/model/Category.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.library.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+
+@Entity
+@Getter
+@Setter
+@RequiredArgsConstructor
+@SQLDelete(sql = "UPDATE categories SET is_deleted = TRUE WHERE id =?")
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false, unique = true)
+    private String name;
+    private String description;
+    @Column(nullable = false)
+    private boolean isDeleted = false;
+
+    public Category(Long id) {
+        this.id = id;
+    }

```

Comment:
Cannot see sense to create constructor for single usage in mapper.
Use there AfterMapping or any other approach that will allow to use no-args constructor + setId()
Having redundant constructor is not better than have 3 additional lines of code in mapper